vllm/config/scheduler.py

```diff
@@ -130,11 +130,12 @@ class SchedulerConfig:
     like full attention and sliding window attention.
     """
 
-    async_scheduling: bool = False
-    """EXPERIMENTAL: If set to True, perform async scheduling. This may help
-    reduce the CPU overheads, leading to better latency and throughput. However,
-    async scheduling is currently not supported with some features such as
-    structured outputs, speculative decoding, and pipeline parallelism.
+    async_scheduling: SkipValidation[bool] = None  # type: ignore
```

Comment:
Once https://github.com/vllm-project/vllm/pull/26519 merges, this can be updated to be 

```suggestion
    async_scheduling: bool = Field(default=None)
```

And `"async_scheduling"` can be added to `_skip_none_validation`

Reply:
OK I updated this, though tbh I'm not sure I understand why it's better! (I think the `#type: ignore` could have been removed anyhow..